pkg/nvml/nvml.go

```diff
@@ -1224,6 +1224,7 @@ func nvmlDeviceGetMPSComputeRunningProcesses_v3(nvmlDevice nvmlDevice, InfoCount
 // nvmlDeviceGetRunningProcessDetailList function as declared in nvml/nvml.h
 func nvmlDeviceGetRunningProcessDetailList(nvmlDevice nvmlDevice, Plist *ProcessDetailList) Return {
 	cnvmlDevice, _ := *(*C.nvmlDevice_t)(unsafe.Pointer(&nvmlDevice)), cgoAllocsUnknown
+	Plist.Version = C.nvmlProcessDetailList_v1
```

Comment:
This should not be set here. This is generated code. It needs to be set in the wrapper passed into this function.

Reply:
Is it possible to add logic to set the version in file "device.go"? I noticed from nvml.h that nvmlProcessDetailList_v1 is actually calculated by the macro definition (size_of(nvmlProcessDetailList_v1_t) | (1 << 24)), and it looks like we need to define some constants to hold such kind of values.

Reply:
I imagine it can be done the same as here: https://github.com/NVIDIA/go-nvml/blob/main/pkg/nvml/device.go#L826

Reply:
Ingenious way, I reworked my submission as in [code](https://github.com/NVIDIA/go-nvml/blob/main/pkg/nvml/device.go#L826) :-)